pkgs/top-level/all-packages.nix

```diff
@@ -6485,7 +6485,7 @@ with pkgs;
 
   circleci-cli = callPackage ../development/tools/misc/circleci-cli { };
 
-  circus = callPackage ../tools/networking/circus { };
+  circus = with python3Packages; toPythonApplication circus;
```

Comment:
no need to change: but maybe someone can clarify why not `python3Packages.toPythonApplication circus`

Reply:
Thanks for reviewing this PR.
Without the `with` statement, it should be `python3Packages.toPythonApplication python3Packages.circus`.

Reply:
ah yes, indeed that makes sense